Sources/MongoSwift/WriteConcern.swift

```diff
@@ -145,7 +145,10 @@ public class WriteConcern: Codable {
     public func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
         try container.encodeIfPresent(self.w, forKey: .w)
-        try container.encodeIfPresent(self.wtimeoutMS, forKey: .wtimeout)
+        if let wtimeout = self.wtimeoutMS {
+            // TODO: change wtimeout to Int64 once tix SWIFT-395 gets fixed
+            try container.encode(Int32(wtimeout), forKey: .wtimeout)
```

Comment:
I think rather than doing `Int32(wtimeout)`, we should use `Int32(truncatingIfNeeded: wtimeout)`. (We already do this in the initializer, see line 103).

With the current approach, the driver will crash if we pass in an Int64 that is too big. With the second one we will just round down.